### ISSUE TITLE
Cheatsheet display style

### ISSUE THREAD

**Issue by vithd** [ionic-team/ionicons#498]
Hi ionic-team, I think the current cheatsheet displaying style is very long and hard to follow what group of icon to search. Instead of showing all the size which is customizable, we should show only one big icon for each and list in grid style just like other font-based icons.

For myself, I have to inject some CSS (Stylebot) to reduce the amount of displaying icons multiply by 10!

```css
.icon-row {
    display: inline-block;
}

.usage .name {
    width: auto;
}

.step:not(.size-64),
.usage label,
.usage input:not(.name),
.usage span  {
    display: none;
}
```

**Comment by brandyscarney** [comment]
Thanks for the issue and sorry for the delay in responding. The documentation has been updated since this was created so this code is no longer changing anything. If you are still having issues with the [current documentation](https://ionic.io/ionicons) or [cheatsheet](https://unpkg.com/ionicons@5.5.2/dist/cheatsheet.html) please let me know, thanks!
